app/workers/scheduled_publisher.rb

```diff
@@ -2,10 +2,10 @@ class ScheduledPublisher
   include Sidekiq::Worker
 
   # 5 retries over 10 mins
-  sidekiq_options :retry => 5
+  sidekiq_options retry: 5
   sidekiq_retry_in do |count|
     # 16s, 31s, 96s, 271s, 640s
-    count ** 4 + 15
+    count**4 + 15
```

Comment:
This was one of the automatic fixes. That is the preferred way according to https://github.com/bbatsov/ruby-style-guide#spaces-operators